src/components/fact-panel/NumericalStats.svelte

```diff
@@ -0,0 +1,103 @@
+<script lang="ts">
+    import ExportFactButton from '../export-code/ExportFactButton.svelte';
+    import type { IQuantMeta } from '../../common/exchangeInterfaces';
+    import DisplayFact from './DisplayFact.svelte';
+    import { formatSort, formatInteger } from '../utils/formatters';
+
+    export let dfName: string;
+    export let colName: string;
+    export let isIndex = false;
+    export let stats: IQuantMeta;
+
+    function formatNumberString(
+        n_zero: number,
+        n_positive: number,
+        n_negative: number
+    ) {
+        if (n_zero === 0 && n_positive === 0 && n_negative === 0) {
+            return 'No values.';
+        } else if (n_zero === 0 && n_positive === 0) {
+            return 'All values are negative.';
+        } else if (n_zero === 0 && n_negative === 0) {
+            return 'All values are positive.';
+        } else if (n_positive === 0 && n_negative === 0) {
+            return 'All values are zero.';
+        } else if (n_zero === 0) {
+            return `There are ${formatInteger(
+                n_positive
+            )} positive values and ${formatInteger(
+                n_negative
+            )} negative values.`;
+        } else if (n_positive === 0) {
+            return `There are ${formatInteger(
+                n_zero
+            )} zero values and ${formatInteger(n_negative)} negative values.`;
+        } else if (n_negative === 0) {
+            return `There are ${formatInteger(
+                n_zero
+            )} zero values and ${formatInteger(n_positive)} positive values.`;
+        } else {
+            return `There are ${formatInteger(
+                n_zero
+            )} zero values, ${n_positive} positive values, and ${formatInteger(
+                n_negative
+            )} negative values.`;
+        }
+    }
+</script>
+
+<ul class="list-disc pt-1 text-gray-600">
```

Comment:
We probably want to change this to have icons for each fact soon so it looks nicer than a list